app/src/main/kotlin/com/simplemobiletools/dialer/helpers/Constants.kt

```diff
@@ -15,11 +15,14 @@ const val SHOW_TABS = "show_tabs"
 const val FAVORITES_CONTACTS_ORDER = "favorites_contacts_order"
 const val FAVORITES_CUSTOM_ORDER_SELECTED = "favorites_custom_order_selected"
 const val WAS_OVERLAY_SNACKBAR_CONFIRMED = "was_overlay_snackbar_confirmed"
-
+const val DIALPAD_VIBRATION = "touch_vibration"
```

Comment:
rename touch_vibration to dialpad_vibration or so, to have constant called the same as the actual value